docs/data/destinations/iterable.md

```diff
@@ -64,6 +64,10 @@ Under **Send Users**, make sure the toggle is enabled ("Users are sent to Iterab
 
 (optional) In **Select additional properties**, select any more user properties you want to send to Iterable. If you don't select any properties here, Amplitude doesn't send any. These properties are sent to Iterable as [Iterable data fields](https://support.iterable.com/hc/en-us/articles/208183076-Field-Data-Types). _Transformed user properties aren't supported._
 
+!!!note "User Forwarding Volumes"
```

Comment:
nit: This could be better maintained as a shared "include". For the destination specific part, if you move it to the end, the rest can be reused.

Example here:
https://github.com/Amplitude-Developer-Docs/amplitude-dev-center/blob/main/docs/data/sdks/android-kotlin/index.md?plain=1#L16-L17

Reply:
Thank for the tip Justin. That's very useful. I think in this case we may want even more destination specific wording coming up so it might not be the best usecase for an include.
I'll merge for now but keep that in mind for the next update.